build-config/checkstyle-suppressions.xml

```diff
@@ -11,4 +11,6 @@
               files="InjectedTest.java"/>
     <suppress checks=".+"
               files="[/\\]generated-sources[/\\]"/>
+    <suppress checks="FileLength"
```

Comment:
https://github.com/jenkinsci/gerrit-trigger-plugin/pull/392#discussion_r274904246

Reply:
I don't mind refactoring - my plan is to move that parts of that subclass logic to it's own file and make it protected. Any concerns with that?

Reply:
Went ahead and did this.